docs/dev/tutorials/custom-paymaster-tutorial.md

```diff
@@ -400,6 +389,12 @@ Balance of the user before mint: 3
 Balance of the user after mint: 102
 ```
 
+## Common errors
+
+If the `use-paymaster.ts` script fails with error `Failed to submit transaction: Failed to validate the transaction. Reason: Validation revert: Paymaster validation error: Failed to transfer funds to the bootloader`, please try sending additional ETH to the paymaster so it has enough funds to pay for the transaction. You can use the [zkSync portal](https://staging-portal.zksync.dev/).
```

Comment:
Remove staging in the link.

> You can use the [zkSync portal](https://staging-portal.zksync.dev/).

I'd say "You can use zkSync Portal", I think we just refer to it as a name

Reply:
fixed in 3940b2be683846ca0039542730511eb23ec262c4